### ISSUE TITLE
Very slow performance on medium to large repositories

### ISSUE THREAD

**Issue by oscardssmith** [waffleio/waffle.io#3186]
I am a developer on https://waffle.io/IQSS/dataverse. We use waffleio a lot, and it is really useful. However I often get unresponsive script warnings, and searching issues (we have about 700 open, 2700 closed) has a considerable amount of lag. Are there plans to fix these problems, or is there something we can do to improve the responsiveness?

**Comment by homeyer** [comment]
Hey @oscardssmith! Try archiving your `Done` column, by clicking the icon in the top right of the column: ![image](https://dl.dropboxusercontent.com/spa/0nz45d6wl4rw5hj/3ei11qx3.png)

**Comment by oscardssmith** [comment]
is there a way to undo archiving?

**Comment by homeyer** [comment]
No, but you can see all of the issues closed in the last 30 days on the throughput page: https://waffle.io/IQSS/dataverse/metrics/throughput

**Comment by sangaline** [comment]
I'm running into this problem as well. I only have ~200-300 issues on the board, but entering text has 5-10 seconds of lag sometimes. Opening an issue can similarly take about 5-10 seconds before the card appears. The repository dropdowns frequently stop working entirely, and I need to refresh the page to get them working again. It's making Waffle pretty much unusable at this point.
